exercises/simple-cipher/src/test/java/SimpleCipherStepTwoTest.java

```diff
@@ -1,79 +1,74 @@
 import org.junit.Before;
-import org.junit.Test;
 import org.junit.Ignore;
+import org.junit.Test;
 
 import static org.junit.Assert.assertEquals;
 
-public class SubstitutionCipherTest {
-
+/**
+ * Step 2: Specify key and use that for shift distance : substitution cipher
+ */
+public class SimpleCipherStepTwoTest {
+    private Cipher cipherWithSetKey;
     private static final String KEY = "abcdefghij";
-    private Cipher cipher;
 
     @Before
     public void setup() {
-        this.cipher = new Cipher(KEY);
+        cipherWithSetKey = new Cipher(KEY);
     }
 
-
+    @Ignore
     @Test
     public void cipherKeepsTheSubmittedKey() {
-        assertEquals(KEY, cipher.getKey());
+        assertEquals(KEY, cipherWithSetKey.getKey());
     }
 
     @Ignore
     @Test
     public void cipherCanEncodeWithGivenKey() {
         String expectedOutput = "abcdefghij";
```

Comment:
Can we come up with a more intention revealing name than `expectedOutput`?  What is the string that's returned from `encode()` represent?